### PR TITLE
logo added

### PR DESCRIPTION
**Pull request by harshs14**

### CODE REVIEW

File: Core/admin.py [hunk 1]
```diff
@@ -1,9 +1,7 @@
 from django.contrib import admin
 from import_export.admin import ImportExportActionModelAdmin
-from .models import Technology, Member, Designation, Project, ContactUs, \
- Registration, Branch, Gender, Year, ContactInfo, Blog, Event, EmailContent, \
- EmailAttachment
-from .forms import RegistrationForm
+from .models import *
```
Comment (bvishal8510) [inline Core/admin.py]: specify the names of functions imported as it was before

File: Core/admin.py [hunk 2]
```diff
@@ -1,9 +1,7 @@
...
+from .forms import *
```
Comment (bvishal8510) [inline Core/admin.py]: here also

File: Core/models.py [hunk 3]
```diff
@@ -104,6 +103,8 @@ class Member(models.Model):
     github = models.CharField(max_length=225, null=True, default=None)
     linkedin = models.CharField(max_length=225, null=True, default=None)
 
+    # message = models.TextField(null=True, blank=True)
```
Comment (bvishal8510) [inline Core/models.py]: remove this

File: Core/templates/alumni_detail.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
+
```
Comment (bvishal8510) [inline Core/templates/alumni_detail.html]: do only required changes in PR. So delete this

File: Core/urls.py [hunk 5]
```diff
@@ -1,12 +1,13 @@
-from .views import IndexView, RegistrationView, BlogView, BlogDetailView, SaveContactView
+from .views import *
```
Comment (bvishal8510) [inline Core/urls.py:1]: this too

File: Core/admin.py (hunk 1 above)
Comment (harshs14) [reply Core/admin.py]: @bvishal8510 sir it will automatically import all the functions and reduces code

File: Core/views.py [hunk 6]
```diff
@@ -54,6 +52,20 @@ def get_context_data(self, **kwargs):
         return context
 
 
+# class AlumniDetailView(View):
```
Comment (bvishal8510) [inline Core/views.py]: remove this

File: Core/models.py (hunk 3 above)
Comment (harshs14) [reply Core/models.py]: @bvishal8510 sir it is the code for alumni section and as designs are not ready, I have commented the entire code for it

File: Core/urls.py [hunk 7]
```diff
@@ -1,12 +1,13 @@
...
 from django.conf.urls import url
 
 urlpatterns = [
     url(r'^$', IndexView.as_view(), name='home'),
     url(r'^savecontact$', SaveContactView.as_view(), name='save_contact'),
     url(r'^register', RegistrationView.as_view(), name='registration'),
     url(r'^blogs', BlogView.as_view(), name='blog'),
-    url(r'^blogs/(?P<pk>\d+)/detail', BlogDetailView.as_view(), name='blog_detail')
+    url(r'^blogs/(?P<pk>\d+)/detail', BlogDetailView.as_view(), name='blog_detail'),
+    # url(r'^alumni/(?P<batch>[0-9]+)/(?P<name>[a-zA-Z]+)/$', AlumniDetailView.as_view(), name='alumni_detail'),
```
Comment (bvishal8510) [inline Core/urls.py]: this too

**Review by bvishal8510 — CHANGES_REQUESTED** [review]
improve commit messages and remove the media files

File: Core/models.py (hunk 3 above)
Comment (bvishal8510) [reply Core/models.py]: but this is not according to PEP8

Comment (bvishal8510) [reply Core/models.py]: So remove it. A PR should only contain relevant changes.

File: Core/admin.py (hunk 1 above)
Comment (bvishal8510) [reply Core/admin.py]: But this is not according to PEP8

**Comment by harshs14** [comment]
@bvishal8510 pls review changes

File: IceCream/settings/base.py [hunk 8]
```diff
@@ -124,15 +124,15 @@
...
+RECAPTCHA_PUBLIC_KEY = '6LcZyJoUAAAAAK_AGiNm_CxiA8RnmYjmDJa-muzd'
 
 
 EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
 EMAIL_HOST = 'smtp.gmail.com'
 EMAIL_PORT = 587
-EMAIL_HOST_USER = os.environ['SENDER_EMAIL']
-EMAIL_HOST_PASSWORD = os.environ['EMAIL_PASSWORD']
-RECEIVER_EMAIL = os.environ['RECEIVER_EMAIL']
+EMAIL_HOST_USER = 'onlineauction.si18@gmail.com'
```
Comment (bvishal8510) [inline IceCream/settings/base.py]: should be taken from the environment variables and remove media files

**Comment by harshs14** [comment]
@bvishal8510 sir all changes are done, pls review

**Comment by bvishal8510** [comment]
Squash all commits into one

**Comment by harshs14** [comment]
@bvishal8510 sir all 3 commits squashed

**Review by bvishal8510 — APPROVED** [review]

**Pull request merged**